lib/muchos/config/azure.py

```diff
@@ -297,15 +297,15 @@ def azure_proxy_host(self):
         return self.get("azure", "azure_proxy_host")
 
     @ansible_host_var
-    @default(None)
+    @default("Standard_D8s_v3")
     def azure_proxy_host_vm_sku(self):
         return self.get("azure", "azure_proxy_host_vm_sku")
```

Comment:
Do we not want to add this property to the `muchos.props` file?

Reply:
Done, addressed in [17babc8](https://github.com/apache/fluo-muchos/pull/397/commits/17babc80c41d917735188f74aed8e9ebad10a79e). I am going to resolve this conversation as I know you are out of office currently and we chatted offline about this.